docs/CNAME

```diff
@@ -0,0 +1 @@
+prysmaticlabs.com
```

Comment:
I think you need to delete the old CNAME since this directory will be the new root